functests/ipv6/ipv6.go

```diff
@@ -48,35 +51,62 @@ var _ = Describe("sriov", func() {
 	})
 
 	var _ = Context("IPv6 configured secondary interfaces on pods", func() {
-		It("Should be able to ping each other", func() {
-			podDefinition := dualStackTestPod(pingedPodName, []string{"/bin/bash", "-c", "--"}, []string{"while true; do sleep 300000; done;"}, nodeSelector)
-			_, err := client.Client.Pods(testNamespace).Create(podDefinition)
-			Eventually(func() k8sv1.PodPhase {
-				runningPod, err := client.Client.Pods(testNamespace).Get(pingedPodName, metav1.GetOptions{})
-				Expect(err).ToNot(HaveOccurred())
-				return runningPod.Status.Phase
-			}, 3*time.Minute, 1*time.Second).Should(Equal(k8sv1.PodRunning))
-			pod, err := client.Client.Pods(testNamespace).Get(pingedPodName, metav1.GetOptions{})
-			var nets []Network
-			err = json.Unmarshal([]byte(pod.ObjectMeta.Annotations["k8s.v1.cni.cncf.io/networks-status"]), &nets)
-			Expect(err).ToNot(HaveOccurred())
-			for _, net := range nets {
-				if net.Interface != sriovInterfaceName {
-					continue
-				}
-				for i, ip := range net.Ips {
-					pingPod(ip, i, nodeSelector)
-				}
-				return
+		It("Should be able to ping each other in a dual stack configuration", func() {
```

Comment:
Would it make sense to have this as a table https://onsi.github.io/ginkgo/#table-driven-tests ?
(re-cycling @SchSeba tip :-) )

Reply:
Done